images/icons/visualwebsitepptimizer.svg

```diff
@@ -0,0 +1 @@
+<svg width="82" height="28" viewBox="0 0 82 28" alt="Visual Website Optimizer" class="top-vwo-logo valign-middle" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M66.822 0c7.91 0 14.32 6.182 14.32 13.808 0 7.626-6.41 13.808-14.32 13.808-6.66 0-12.26-4.386-13.86-10.326l1.27.795.658-.977c1.486 5.004 6.265 8.667 11.932 8.667 6.854 0 12.41-5.358 12.41-11.967 0-6.61-5.556-11.967-12.41-11.967-.586 0-1.163.04-1.728.116l1.2-1.782-.247-.154c.257-.013.515-.02.775-.02z" fill="#BF3078"></path><path fill="#802050" d="M52.503 0h14.32L47.73 27.616 40.57 17.26l-7.16 10.356-7.24-10.473 1.133-1.68 6.108 8.834 5.572-8.058 1.59-2.3 1.587 2.3 5.57 8.057L63.255 1.84H36.476L37.716 0"></path><path d="M19.092 27.616L0 0h38.184L19.092 27.616zm0-3.32L34.617 1.842H3.567l15.525 22.457z" fill="#26134D"></path></g> </svg>
```

Comment:
Maybe we can replace it with this a bit cleaner svg:
```svg
<svg viewBox="0 0 279 96" xmlns="http://www.w3.org/2000/svg">
    <!-- Source: https://commons.wikimedia.org/wiki/File:VWO-Logo.svg -->
    <!-- CC BY-SA 4.0 by Nitika.g4 -->
    <path d="M229.8 0A48.6 48.6 0 0 1 279 48c0 26.5-22 48-49.2 48A49 49 0 0 1 182 60.1l4.4 2.8 2.2-3.4a42.5 42.5 0 0 0 41 30.1A42.1 42.1 0 0 0 272.5 48a42.1 42.1 0 0 0-48.6-41.2L228 .6l-.8-.5 2.7-.1"
          fill="#BF3078"/>
    <path d="M180.5 0h49.3L164 96l-24.6-36-24.6 36-25-36.4 4-5.9 21 30.8 19.1-28 5.5-8 5.5 8 19.1 28 53.4-78.1h-92l4.2-6.4h50.8"
          fill="#802050"/>
    <path d="M65.7 96L0 0h131.3L65.6 96m0-11.5L119 6.4H12.3l53.3 78" fill="#26134D"/>
</svg>
```

Reply:
Thank, you, I have updated my commit.